### DISCUSSION TITLE
[BUG]Elsa 3.1.3 uses a visual designer to implement workflow approval. When a node approval is rejected, the process cannot return to the previous node or the original node?

### DISCUSSION THREAD

**Discussion by captaris** [elsa-workflows/elsa-core#5845] [Q&A]
![Reject](https://github.com/user-attachments/assets/a5f825c1-43ce-417f-a9fd-b0c8b906b01b)

**Comment by sfmskywalker** [comment]
You can connect the Reject outcome to a previous node of your choosing. 
That being said, the Fork activity will execute **both** Reject and Approval, so your workflow will probably not accomplish your goal. Instead, you might want to consider using the Switch (flow) activity.
